modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -75,7 +75,7 @@
 public class CellAdapter
     implements Cell, CellEventListener, CellEndpoint
 {
-    private static final Logger _log =
+    private static final Logger logger =
```

Comment:
https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case (3rd comment) shall we change it to uppercase anyway?